src/Cake.Core.Tests/Unit/Scripting/Processors/LoadDirectiveProcessorTests.cs

```diff
@@ -137,5 +138,43 @@ public void Should_Insert_Line_Directives_When_Processing_Load_Directives()
             Assert.Equal(result.Lines[11], "// #l b.cake");
             Assert.Equal(result.Lines[12], "int y=2;");
         }
+
+        [Theory]
+        [InlineData("#load \"/utils.cake\"")]
+        [InlineData("#load \"local:?path=/utils.cake\"")]
+        public void Should_Process_AbsolutePath_Script_Reference_Found_In_Source(string source)
```

Comment:
Can we also add tests for relative file paths?

Reply:
Isn't `Should_Process_Single_Script_Reference_Found_In_Source` and `Should_Process_Single_Script_Reference_With_Spaces_In_File_Name_Found_In_Source` also test the loading of relative pathes? That's why i add tests only with absolute path. If you think that it would be useful to test relative pathes in these new tests, i can add them. But then i need to rename the methods because now they have `AbsolutePath` part in their name.

Reply:
I am thinking how to do this correctly. All the other tests only works for one script path. Because of the second `GivenScriptExist` call eg: `fixture.GivenScriptExist("c:/utils.cake", "Console.WriteLine();");`

If i add relative path to this test, then first i should parse the directive and get the path from it, just to set up the test context. Do you have any idea how should i do that?

Reply:
Easiest would just be to duplicate the two tests you already written and change to relative paths.

Reply:
Yeah, first i tried to solve it in one test. But splitting it into two is straightforward. :)

Reply:
Done